everestrs/everestrs/build.rs

```diff
@@ -24,8 +28,19 @@ fn find_everest_workspace_root() -> PathBuf {
 /// Returns the Libraries path if this is a standalone build of everest-framework or None if it is
 /// not.
 fn find_libs_in_everest_framework(root: &Path) -> Option<Libraries> {
-    let everestrs_sys = root.join("everest-framework/build/everestrs/libeverestrs_sys.a");
-    let framework = root.join("everest-framework/build/lib/libframework.so");
+    let (everestrs_sys, framework) =
+        if let Ok(everest_framework_dir) = env::var("EVEREST_RS_FRAMEWORK_BINARY_LOCATION") {
```

Comment:
Compare the naming:
`if let Ok(everest_framework_dir) = env::var("EVEREST_RS_FRAMEWORK_SOURCE_LOCATION") {`
and
`if let Ok(everest_framework_dir) = env::var("EVEREST_RS_FRAMEWORK_BINARY_LOCATION") {`

Reply:
we do need the binary location here, not the source location, which are usually not identical